backend/tests/unit/app/core/test_logging.py

```
@@ -0,0 +1,128 @@
+            test_message = "Test log message"
+            logger.info(test_message)
+            
+            # Assert
+            log_output = log_stream.getvalue()
+            assert test_message in log_output
+            # Human-readable format should include timestamp and level
+            assert "INFO" in log_output
+            # Should NOT be JSON
+            try:
+                json.loads(log_output.strip())
+                assert False, "Development logs should not be JSON formatted"
```

Ruff (0.13.1)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)